protocol/activity_center.go

```diff
@@ -65,6 +65,9 @@ type ActivityCenterNotification struct {
 	Deleted                   bool                           `json:"deleted"`
 	Accepted                  bool                           `json:"accepted"`
 	ContactVerificationStatus verification.RequestStatus     `json:"contactVerificationStatus"`
+	//Used for synchronization. Each update should increment the clock.
+	//The value should represent the time when the update occurred.
+	Clock uint64 `json:"clock"`
```

Comment:
Suggestion: the name could be more specific. For instance `UpdatedAt`. `UpdatedAt` is a common convention to mean the column is a timestamp and the name more precisely conveys the purpose of the column. If possible, consider renaming the DB column as well.

Another point is that we already have a `timestamp` column in the table `activity_center_notifications`, so I'm thinking here, could we use it instead of adding another one? The name `timestamp` is too vague (timestamp of what?), so I'm not sure if what I'm suggesting even makes sense.

Reply:
I have also thought about the naming issue of clock before. Its meaning is not accurate enough, but almost all synchronization implementations use clock. Would you still use UpdatedAt after knowing this situation? I am open-minded about this issue :). Timestamp cannot be used as a clock because it only records the time when the record was first inserted. If it is also used to record the last updated time, it seems inappropriate.

Reply:
@qfrank, I think nobody will know what just "clock" means without jumping to the struct field documentation. Even though there are other places in the code using the word clock, many are accompanied by something else to convey its purpose, like `LastUpdateClock`, `ReadMessageAtClockValue`, `NextClockAndTimestamp`, so I think we could do something similar.

> Timestamp cannot be used as a clock because it only records the time when the record was first inserted. If it is also used to record the last updated time, it seems inappropriate.

Makes complete sense.

Reply:
I'll rename it to `updated_at` 🫶